docker-app/qfieldcloud/core/admin.py

```diff
@@ -147,7 +147,7 @@ def admin_urlname_by_obj(value, arg):
 )
 
 
-class EmailAddressAdmin(EmailAddressAdmin):
+class ExtendedEmailAddressAdmin(EmailAddressAdmin):
```

Comment:
A bit picky detail, but we do the other way around so far by renaming the import:

```suggestion
class EmailAddressAdmin(EmailAddressAdminBase):
```